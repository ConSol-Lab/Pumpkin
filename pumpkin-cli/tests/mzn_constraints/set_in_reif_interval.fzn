var bool : reif :: output_var;

var 1..7 : x ::output_var;

constraint set_in_reif(x, 1..3, reif);

solve satisfy;