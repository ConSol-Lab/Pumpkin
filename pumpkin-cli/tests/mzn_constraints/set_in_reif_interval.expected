reif = false;
x = 4;
----------
reif = false;
x = 5;
----------
reif = false;
x = 6;
----------
reif = false;
x = 7;
----------
reif = true;
x = 1;
----------
reif = true;
x = 2;
----------
reif = true;
x = 3;
----------
==========